### PR TITLE
Change GeoJSON geometry when tag suggests area feature

### PR DESCRIPTION
**Pull request by vgeorge**
See #128. 

@geohacker I suggest testing with preset "Apartment Complex".

### CODE REVIEW

File: app/screens/Features/AddFeatureDetail.js [hunk 1]
```diff
@@ -26,13 +26,13 @@ import { getParentPreset } from '../../utils/get-parent-preset'
 import { colors } from '../../style/variables'
 import { KeyboardAwareScrollView } from 'react-native-keyboard-aware-scroll-view'
 import { modes } from '../../utils/map-modes'
+import { osmTagSuggestingArea } from '../../utils/get-area-tags'
```
Comment (geohacker) [inline app/screens/Features/AddFeatureDetail.js:29]: utils/get-area-tags is missing I think

File: app/utils/oms-tag-suggesting-area.js [hunk 2]
```diff
@@ -0,0 +1,98 @@
+import { presets as allPresets } from '../presets/presets.json'
```
Comment (geohacker) [inline app/utils/oms-tag-suggesting-area.js:1]: The file name doesn't match the import. Typo in the filename (oms -> osm)

**Comment by geohacker** [comment]
@vgeorge this looks great, thank you! One thing we should probably confirm: when a feature is considered to be an area, do we now add the `area=yes` tag? Does iD do that? If it does, we should add that to the properties.

**Comment by vgeorge** [comment]
@geohacker, we don't. I did two tests in iD: added a feature by selecting "Area" button at the top and selected a "area" preset on a closed way, none of them added `area=yes`. If I apply this tag manually to a feature in iD, it will be rendered differently, but presets doesn't seem to change it. It seems it uses `area=*` exclusively for rendering purposes.

**Pull request merged**